### PR TITLE
[Wallet] Fix default background color for screens

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

- This fixes the incorrect gray color in the CELO education screen
- Also stop redefining the background color for screens when unnecessary

### Tested

**before/after**

### Related issues

- #4415

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/navigator/NavigatorWrapper.tsx [hunk 1]
```diff
@@ -32,6 +33,15 @@ const getActiveRouteName = (state: NavigationState) => {
...
 
+// Global app them used by react-navigation
+const AppTheme = {
+  ...DefaultTheme,
+  colors: {
+    ...DefaultTheme.colors,
+    background: colors.light,
+  },
+}
+
```
Comment (jeanregisser) [inline packages/mobile/src/navigator/NavigatorWrapper.tsx:44]: This is where we define the default background color for screens.

File: packages/mobile/src/home/WalletHome.tsx [hunk 2]
```diff
@@ -171,7 +171,7 @@ export class WalletHome extends React.Component<Props, State> {
       <RefreshControl
         refreshing={this.props.loading}
         onRefresh={this.onRefresh}
-        colors={[colors.celoGreen]}
+        colors={[colors.greenUI]}
```
Comment (aaronmgdr) [inline packages/mobile/src/home/WalletHome.tsx:174]: nice

File: packages/mobile/src/navigator/NavigatorWrapper.tsx (hunk 1 above)
Comment (aaronmgdr) [reply packages/mobile/src/navigator/NavigatorWrapper.tsx:44]: cool.

also just noticed you can comment on multiple lines. nice

File: packages/mobile/src/account/Profile.tsx [hunk 3]
```diff
@@ -69,18 +71,6 @@ const styles = StyleSheet.create({
     flexDirection: 'column',
     alignItems: 'center',
   },
-  underlinedBox: {
-    borderTopWidth: 1,
-    borderColor: '#EEEEEE',
-  },
-  scrollView: {
```
Comment (aaronmgdr) [inline packages/mobile/src/account/Profile.tsx:76]: were these unused?

Comment (jeanregisser) [reply packages/mobile/src/account/Profile.tsx:76]: Yep

**Review by aaronmgdr — APPROVED** [review]
Assuming that this works like you say it does it looks great! 

(ps i think you have some react-components package tests to fix)

**Pull request merged**